integration_test/third_party_apps_data/applications/flink/metadata.yaml

```diff
@@ -1,10 +1,255 @@
 public_url: "https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/flink"
```

Comment:
Can we get rid of the public_url? See https://github.com/GoogleCloudPlatform/ops-agent/pull/632#discussion_r905486897

It will be set by the post agent release job that will run